include/supercluster.hpp

```diff
@@ -243,6 +243,16 @@ class Supercluster {
         }
 
         Zoom(Zoom &previous, const double r, const std::uint8_t zoom) {
+
+            // Zoom encoded in the lower 5 bits
+            bool valid_zoom = (((zoom + 1) & 0b11111) == (zoom + 1));
+#ifdef NDEBUG
+            if (!valid_zoom)
+                return;
+#else
+            assert(valid_zoom);
+#endif
```

Comment:
I think the code will panic even if you return early, so I'd maybe simplify all that to just:

```cpp
assert((zoom + 1) & 0b11111 == zoom + 1);
```

Reply:
There's no need to only `assert` in debug mode. Asserts are omitted in Release mode anyway. They are not a safe coding practice, so even in Debug mode, we should take some action.

Additionally, let's use braces even for single line blocks.

Reply:
Sorry, I'm confused. Are you suggesting that we remove the assert completely, and just have the return? Or that we have both without the `ifdef`?

Reply:
The latter